manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml

```diff
@@ -0,0 +1,54 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Sohu.SHPlayer
+PackageVersion: 7.0.6.0
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- 3gp
+- 3gpp
+- asf
+- avi
+- difox
+- f4v
+- flc
+- fli
+- flv
+- ifox
+- m2p
+- m2t
+- m2ts
+- m2v
+- m4v
+- mkv
+- mov
+- mp4
+- mpeg
+- mpg
+- qt
+- ram
+- rm
+- rmvb
+- ts
+- vob
+- webm
+- wmv
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: exe
```

Comment:
```suggestion
  InstallerType: nullsoft
```

Reply:
Although the installer uses switches similar to NSIS, many detectors including 7-zip cannot recognize it as a Nullsoft Installer. So I change to use custom switches.🤔